public/openapi-3-v3.1.json

```diff
@@ -1487,9 +1372,6 @@
           "400": {
             "$ref": "#/components/responses/BadRequest"
           },
-          "401": {
-            "$ref": "#/components/responses/Unauthorized"
-          },
           "404": {
             "$ref": "#/components/responses/NotFound"
           }
```

Comment:
in the end of `/sources/{id}/rhc_connections` and` /rhc_connections/{id}/sources` we have tags with values "sources" and "rhc-connections" ... it causes that both methods are displayed in endpoints related with both of this groups (example on picture) ... but we follow this approach only here and not for others endpoints ... I think, it should be same in whole spec

![image](https://user-images.githubusercontent.com/89980168/172350413-7615676b-873b-4ad6-9c33-6aa5c30f5239.png)

Reply:
All right, I've removed the redundant tags!